lib/ansible/modules/windows/win_domain_group_membership.ps1

```diff
@@ -52,7 +54,17 @@ $members_before = Get-AdGroupMember -Identity $name @extra_args
 $pure_members = [System.Collections.Generic.List`1[String]]@()
 
 foreach ($member in $members) {
-    $group_member = Get-ADObject -Filter "SamAccountName -eq '$member' -and $ad_object_class_filter" -Properties objectSid, sAMAccountName
+    $extra_member_args = @{}
+    if ($member -match "\\"){
+      $extra_member_args.Server = $member.Split("\")[0]
```

Comment:
When making changes to a file please keep similar formatting as the existing code, that includes things like indent size.